android/build.gradle.kts

```diff
@@ -29,6 +29,9 @@ version = libs.versions.libraryVersion.get()
 nexusPublishing {
     repositories {
         sonatype {
+            nexusUrl.set(uri("https://ossrh-staging-api.central.sonatype.com/service/local/"))
```

Comment:
As we can see in the url above, this shows we'll be using the [Portal OSSRH Staging API](https://central.sonatype.org/publish/publish-portal-ossrh-staging-api/), a kind of legacy-gateway to the new Portal API provided by Sonatype as a stopgap until they've produced their own Gradle plugin for publishing to Maven Central via the Central Publishing Portal.

<img width="698" alt="image" src="https://github.com/user-attachments/assets/b8636508-25cf-4ba4-94b3-e65a7a008961" />

...this is preferable to using a third-party plugin for now! Once Sonatype produce their own Gradle plugin, we'll probably want to change this.

For Scala projects, we're now using the new in-built support for the Portal API in `sbt`:

* https://github.com/guardian/gha-scala-library-release-workflow/pull/61